Tools/autotest/arducopter.py

```diff
@@ -5148,6 +5148,31 @@ def test_parameter_validation(self):
         self.drain_mav()
         self.assert_prearm_failure("Check MOT_PWM_MIN/MAX")
 
+    def test_alt_estimate_prearm(self):
+        self.context_push()
+        ex = None
+        try:
+            self.set_parameter("EK2_ALT_SOURCE", 2)
+            self.wait_gps_disable()
+            self.wait_heartbeat()
```

Comment:
This wait_heartbeat() should be in `change_mode()`.

Reply:
done